docker/search/pom.xml

```diff
@@ -0,0 +1,67 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
perhaps we could look at this file and update some versions. Given the old version of solr and java, I don't know how much we can upgrade